server/@types/viewModels/index.d.ts

```diff
@@ -361,6 +361,36 @@ declare module 'viewModels' {
     level: 'ENTRY_LEVEL' | 'LEVEL_1' | 'LEVEL_2' | 'LEVEL_3' | 'LEVEL_4' | 'LEVEL_5' | 'LEVEL_6' | 'LEVEL_7' | 'LEVEL_8'
   }
 
+  export interface Timeline {
+    reference?: string // TODO - Are these required/optional?
```

Comment:
Need to have a look at this, when attempting to pass the 

```
const timeline = await this.educationAndWorkPlanService.getTimeline(prisonNumber, req.user.token)
```

into the `TimelineView` in the `overviewController` 

`const view = new TimelineView(prisonerSummary, timeline)`

It said that the `reference and `prisonNumber` were missing 🤔